content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,131 @@
+# Charte Code de Conduite Contributeurs

```

Comment:
This reads a bit funny. The options are:

1. Not translate "Contributor Covenant" but italicize it, like [Wikipedia does](https://fr.wikipedia.org/wiki/Contributor_Covenant):
    ```suggestion
    # Code de conduite _Contributor Covenant_
    ```
2. Translate it:
    ```suggestion
    # Pacte du/de la contributeur•rice – Code de conduite
    ```
3. Translate it but pluralize to make the inclusive language less painful:
    ```suggestion
    # Pacte des contributeur•rice•s – Code de conduite
    ```
4. Leave it out:
    ```suggestion
    # Code de conduite
    ```